controllers/userController.js

```diff
@@ -101,6 +102,41 @@ router.post('/:id/password', verifyToken, (req, res) => {
 	});
 });
 
+// Changes a don's information (only accessible by dons)
+router.post('/:id/doninfo', verifyToken, (req, res) => {
+	if(req.user.role&1 != 1) {
+		console.error("User "+ req.user._id +"is requesting don info when user is not a don");
+		//requestor is not a don
+		res.status(403);
+	}
+	User.findOne({_id: req.params.id}).then(user => {
+		if(user.role&1 != 1){
+			//user is not a don
+			res.status(403);
```

Comment:
This should probably be a 400 error, not 403. It's not an authentication error, it's a semantically invalid request.